### DISCUSSION TITLE
FEATURE :sparkles: Add transitions while changing the pages.

### DISCUSSION THREAD

**Discussion by shivamnarkar47** [keizerworks/keizer-web#20] [Ideas]
### How about adding a transition to this magnificient style website ? 
Like when I navigate to different pages, it must be like some black overlay transition.
It will look nice on this website.

### Anything else?

For reference something like this : 
https://dorstenlesser.com/

Also there is an option of willing to submit PR ? So for now , I am short on time. But if in future, will definitely help. :dove: 

### Are you willing to submit PR?

No

**Comment by aayushchugh** [comment]
I like this animation, @keizerworks/keizer-web what do you guys say?

**Comment by pranshuj73** [comment]
assign it to me please @shivamnarkar47

**Comment by shivamnarkar47** [comment]
> assign it to me please @shivamnarkar47

I am not the owner, maybe @aayushchugh or the admin will help you.

**Comment by aayushchugh** [comment]
I am converting this to a discussion so that we can get feedback from other team members too

**Comment by aayushchugh** [comment]
@shivamnarkar47 @pranshuj73 This animation will make the whole website client only which is not efficient for SEO so we have decided not to implement this animation

**Reply by pranshuj73** [reply]
thats fair ig, im pretty sure there would've been work arounds but this is understandable

**Reply by aayushchugh** [reply]
If you can share any, we'd love to hear this
